### ISSUE TITLE
[Translation] Chinese(Simplified) Docs Translation in Progress

### ISSUE THREAD

**Issue by StephenArk30** [taichi-dev/taichi#735]
If anyone is interested in contributing to **Chinese (Simplified)** Docs with me, please let me know, so that we can work together and speed things up! I will mainly follow the order to continue, if you are willing to contributing, you are always welcome to pick up any page you like!

简体中文文档正在翻译中，欢迎有兴趣加入的同学一起翻译。如果需要联系我可以发邮件到stephenark@163.com。

I copied [your issue](https://github.com/felangel/bloc/issues/909) because I didn't know the format, sorry. @jakecastelli

**Comment by yuanming-hu** [comment]
Welcome @StephenArk30 and thank you for contributing! This is very meaningful and will benefit a lot of Chinese Taichi users!

Please feel free to open a draft PR so that you get early feedback. Please also check out https://taichi.readthedocs.io/en/latest/contributor_guide.html.

谢谢！

**Comment by yuanming-hu** [comment]
@StephenArk30 一些罕见词的翻译，或许会有帮助 :-)

Taichi: Taichi（太极）[我们可以保持Taichi的名字，毕竟大部分编程语言都在用英文名字，比如说Python不会被翻译成蟒蛇；不过第一次出现的时候可以再括号里面标注一下“太极”，毕竟中文环境下“图太极”更容易发音。]

Productivity：生产力
Performance：性能
Portability：可移植性
Spatially sparse computation：空间稀疏计算
Differentiable programming：可微编程
Metaprogramming：元编程

Decouple computation from data structures：解耦计算与数据结构
Domain-specific compiler optimizations：领域特定编译器优化
Megakernels：大内核
Two-scale automatic differentiation：双尺度自动微分
Embedding in Python：嵌入在Python

embedded domain-specific language：嵌入式领域特定语言
tensor：张量（多维数组）
Functions and kernels：函数与核心
scope：作用域
active element：活跃元素
range-for：范围for循环
struct-for：结构for循环
plain indexing syntax：普通下标语法(如x[i, j])

**Comment by archibate** [comment]
一些建议:

> 嵌入在Python

嵌入Python，入已是介词

> 核心

内核，或者直接kernel，来自CUDA的人们应该熟悉

> 活跃元素

活动(active)元素，第一次出现时，应标上相应英文，以和API中的`is_active`对照方便理解

> 范围for循环

区间for循环，我看到python的range常常这样翻译

**Comment by yuanming-hu** [comment]
@StephenArk30 `requests` has multi-lagnuage doc: https://requests.readthedocs.io/zh_CN/latest/

Maybe we should adopt their solution.

**Comment by StephenArk30** [comment]
经过一些研究，一方面为了方便以后其它语言的翻译，一方面考虑到原仓库体积的问题，我们决定参考python官方文档翻译的流程，另起炉灶开一个仓库 : )

仓库地址：https://github.com/StephenArk30/taichi-docs-zh-cn
参考了python官方的做法：https://www.python.org/dev/peps/pep-0545/
python文档法语翻译仓库：https://github.com/python/python-docs-fr

另外，或许我们也可以考虑给taichi标上版本号，像[cpython](https://github.com/python/cpython)的2.7、3.7 branch。

**Comment by yuanming-hu** [comment]
Sounds good. It seems that `requests` is also using separate repos for their doc: https://github.com/requests/requests-docs-cn



> 另外，或许我们也可以考虑给taichi标上版本号，像[cpython](https://github.com/python/cpython)的2.7、3.7 branch。

Right. Currently, Taichi is growing fast, so I think to have a single version of the documentation that reflects the **latest** taichi would be good enough. In the future, we can have branches for v1.0, v1.1 etc.

**Comment by jackalcooper** [comment]
I would like to make a pitch for this platform: https://crowdin.com/

**Comment by yuanming-hu** [comment]
> I would like to make a pitch for this platform: https://crowdin.com/ . You can easily organize multi-language documentations on it. For instance https://crowdin.com/project/bucklescript/en#

Thanks - that looks cool but let's stick to what we have for now.

@StephenArk30 Thanks to your hard work, the ["Hello world" part is now mostly available in Chinese](https://taichi-docs-zh-cn.readthedocs.io/zh_CN/latest/hello.html#). Do you mind transferring the ownership of that repo to `taichi-dev`? This will bring more people's attention to what you are doing. Of course, you will continue to be the admin of the Chinese document repo.

**Comment by StephenArk30** [comment]
Right, I have transferred the ownership. Doc url can be moved to ``taichi.readthedocs.io/cn/``.

**Comment by yuanming-hu** [comment]
Hi @StephenArk30 @archibate thanks for your great contribution to the Chinese documentation! Could you email me your readthedocs account (yuanmhu a_t gmail d_o_t com)? I can add you guys to the maintainer lists of the taichi documentation so that you can have access to the translation settings.

**Comment by yuanming-hu** [comment]
@StephenArk30 I added your account to the Taichi readthedocs project yesterday. @archibate Could you email me your account as well? Thanks!

**Comment by jerviscui** [comment]
The Chinese document link is not working.
